config-templates/aws/aws-ebs-csi-driver/0.9.14/metadata.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "0.9.14",
+  "enabled": "true",
+  "description": "AWS EBS CSI Driver version 0.9.14",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "aws"
+    },
+    {
+      "name": "offering-name",
+      "value": "aws-ebs-csi-driver"
+    },
+    {
+      "name": "provider-url",
+      "value": "https://console.aws.amazon.com/support/home#/"
+    },
+    {
+      "name": "support-msg",
+      "value": "For support contact AWS"
+    },
+    {
+      "name": "provider-logo",
+      "value": "/aws-ebs-logo"
```

Comment:
Create a `logo` dir and copy `aws-ebs-csi-driver.png` under `logo`  refer to  https://github.com/IBM/ibm-satellite-storage/tree/develop/config-templates/aws/aws-ebs-csi-driver/0.8.2/logo
and update to `"value": "logo/aws-ebs-csi-driver.png"`

Reply:
Addressed